tools/downloader/pytorch_models_conversion.md

```diff
@@ -0,0 +1,88 @@
+# Conversion pretrained models from PyTorch to ONNX
```

Comment:
Ok, prepare to release this featurelater, but if you insist, I'll do it in this patch.
So, the main issue is that any model can be added to conversion without any changes in script. I tested it on few models and it works.

Reply:
> Ok, prepare to release this featurelater, but if you insist, I'll do it in this patch.

Wait, what feature? I'm not asking for any new features. I'm actually suggesting the opposite - I don't think the conversion script should be a feature by itself. IMO, it should just be an implementation detail of the model converter.

Reply:
Yes, but you pointed on very weak part of converter. So, with your help I fixed it, making it able to solve more common problem.

Reply:
OK, I like what you did wrt arbitrary model support, but I think we should still keep this script internal for now. I don't like the idea of exposing it right after we add it, because as we add more models, we might want to change it in an incompatible way. Let's give it some time to mature.

If you agree, then please remove the separate README and add any relevant parts to the downloader README. If not, then let me know why. :-)